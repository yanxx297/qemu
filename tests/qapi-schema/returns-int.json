# It is okay (although not extensible) to return a non-dictionary
{ 'command': 'okay', 'returns': 'int' }
