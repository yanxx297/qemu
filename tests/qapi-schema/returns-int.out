command guest-get-time None -> int
   gen=True success_response=True
